### PR TITLE
New version: OSGeo.QGIS version 3.38.1 (3.38.1-1)

### PR DESCRIPTION
**Pull request by spectopo**
Created by [🥟 Dumplings](https://github.com/SpecterShell/Dumplings) in workflow run [#7139](https://github.com/SpecterShell/Dumplings/actions/runs/10265329701).

<details>

<summary>Log</summary>

````
Updated: 3.38.0-1 → 3.38.1-1
Submitting WinGet manifests
Creating manifests
Uploading manifests and making commits
Creating a pull request
````

</details>

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/166343)

### CODE REVIEW

**Comment by stephengillie** [comment]
URL: https://qgis.org/downloads/QGIS-OSGeo4W-3.38.1-1.msi 
Status Code: 200

(Automated message - build 887)

File: manifests/o/OSGeo/QGIS/3.38.1/OSGeo.QGIS.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+Moniker: qgis
+Tags:
+- gis
+# ReleaseNotes:
+ReleaseNotesUrl: https://www.qgis.org/en/site/forusers/visualchangelogs.html
+# PurchaseUrl:
+# InstallationNotes:
+Documentations:
+- DocumentLabel: Documentation
+  DocumentUrl: https://www.qgis.org/en/docs/
```
Comment (stephengillie) [inline manifests/o/OSGeo/QGIS/3.38.1/OSGeo.QGIS.locale.en-US.yaml:29]:
```suggestion
#  DocumentUrl: 
```

File: manifests/o/OSGeo/QGIS/3.38.1/OSGeo.QGIS.locale.zh-CN.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+# Moniker:
+Tags:
+- 地理信息系统
+# ReleaseNotes:
+ReleaseNotesUrl: https://www.qgis.org/zh-Hans/site/forusers/visualchangelogs.html
+# PurchaseUrl:
+# InstallationNotes:
+Documentations:
+- DocumentLabel: 文档
+  DocumentUrl: https://www.qgis.org/zh-Hans/docs/
```
Comment (stephengillie) [inline manifests/o/OSGeo/QGIS/3.38.1/OSGeo.QGIS.locale.zh-CN.yaml:29]:
```suggestion
#  DocumentUrl: 
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @spectopo,

It appears that the documentation site is blocking our automated scans.

**Review by ItzLevvie — APPROVED** [review]

**Review by stephengillie — APPROVED** [review]

**Comment by stephengillie** [comment]
This package installs and launches normally on a Windows 10 VM.

**Comment by stephengillie** [comment]
This package installs and launches normally on a Windows 10 VM.

**Pull request merged**